src/System.Data.Common/tests/System/Data/Common/DbExceptionTests.cs

```diff
@@ -8,22 +8,70 @@ namespace System.Data.Common.Tests
 {
     public class DbExceptionTests
     {
+        [Fact]
+        public void DefaultArgumentsShowed()
```

Comment:
Nit: I'm not sure what "Showed" means here.  Maybe "ParameterlessConstructor_UsesDefaults", or something like that?